lldb/NOTICE.TXT

```diff
@@ -0,0 +1,7 @@
+
+This product contains small piece of code to support AIX, taken from netbsd.
+
+  * LICENSE:
+    * lldb/source/Host/common/LICENSE.aix-netbsd.txt (OpenSSL License)
```

Comment:
We'll need to confirm license compatibility for this.

Reply:
And if there is a way not to pull in this code, we're likely going to prefer that.